manheim_cloudmapper/send_email.py

```diff
@@ -0,0 +1,10 @@
+from ses.report import Report
+
+
+def send_email():
+    report = Report()
```

Comment:
Since the variable ``report`` is only used once, please simplify to: ``Report().generate_and_send_email()``